src/test/javascript/spec/component/code-editor/code-editor-ace.component.spec.ts

```diff
@@ -31,10 +29,10 @@ describe('CodeEditorAceComponent', () => {
     let codeEditorRepositoryFileService: CodeEditorRepositoryFileService;
     let loadRepositoryFileStub: SinonStub;
 
-    beforeEach(async () => {
+    beforeEach(() => {
         return TestBed.configureTestingModule({
-            imports: [TranslateModule.forRoot(), ArtemisTestModule, AceEditorModule, TreeviewModule.forRoot(), ArtemisProgrammingManualAssessmentModule],
-            declarations: [CodeEditorAceComponent, MockPipe(ArtemisTranslatePipe)],
+            imports: [ArtemisTestModule, AceEditorModule],
```

Comment:
Can we remove the `AceEditorModule` here too?

Reply:
I tried to do that, but unfortunately it is needed here, I wasn't able to replace it with according MockComponents